external/builder/fixtures_babel/constructors.js

```diff
@@ -0,0 +1,21 @@
+class A {
+  constructor() {
+    console.log("Hi!");
+  }
+}
+
+class B {
+  constructor(x = console.log("Hi!")) {}
+}
+
+class C {
+  constructor({ x }) {}
+}
+
+class D {
+  constructor(x, y, z) {}
+}
+
+class E extends A {
+  constructor() {}
+}
```

Comment:
Can you please also add a couple of test-cases for the situation that prompted the issue, i.e. where the pre-processor is used?
```suggestion
}

class F {
  constructor() {
    if (PDFJSDev.test('TRUE')) {
      var a = 0;
    }
  }
}

class G {
  constructor() {
    if (PDFJSDev.test('FALSE')) {
      var a = 0;
    }
  }
}
```

Reply:
Done